src/coreComponents/physicsSolvers/CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ set( physicsSolvers_headers
      solidMechanics/SolidMechanicsLagrangianFEM.hpp
      solidMechanics/SolidMechanicsLagrangianSSLE.hpp
      solidMechanics/SolidMechanicsLagrangianFEMKernels.hpp
+     solidMechanics/SolidMechanicsEFEMKernels.hpp
+     solidMechanics/EFEMKernelsHelper.hpp
```

Comment:
perhaps files should be named `SolidMechanicsEFEMKernelsHelper.hpp` for consistency.